sdktests/server_side_persistence_base.go

```diff
@@ -143,7 +143,7 @@ func newServerSidePersistentTests(
 }
 
 func (s *ServerSidePersistentTests) Run(t *ldtest.T) {
-	t.Run("uses default prefix", func(t *ldtest.T) {
+	s.runWithEmptyStore(t, "uses default prefix", func(t *ldtest.T) {
```

Comment:
This was an issue I noticed when running the tests against a different local service, so I'm adding that on here.